meta-balena-scarthgap/recipes-kernel/linux/linux-firmware_%.bbappend

```diff
@@ -0,0 +1,17 @@
+inherit balena-linux-firmware
+
+# Cleanup iwlwifi firmware files
+IWLWIFI_PATH = "lib/firmware"
```

Comment:
switching to
`WLWIFI_PATH = "${D}${nonarch_base_libdir}/firmware"`

fixes 

`
Exception: FileNotFoundError: [Errno 2] No such file or directory: 'builds/balena-jetson-orin/build_agx_orin/tmp/work/all-poky-linux/linux-firmware/20230404/usr/lib/firmware'
`